engine_test.go

```diff
@@ -455,8 +505,8 @@ func TestEval_AllowUnknownSelector(t *testing.T) {
 
 func TestRandomExpressions(t *testing.T) {
 	const (
-		size          = 3000000
-		level         = 53
+		size          = 10000000
+		level         = 17
```

Comment:
Because we use int16 as the type of the slice index, and the length of the `bytecode` is 8 times the length of the `nodes`. so the length (_or the complexity_) of the expression is reduced.

https://github.com/larry618/eval/blob/2d5ee1141e64bf9378c6730bf1eae99811ba1fa6/compile.go#L144-L153